pyartcd/pyartcd/jenkins.py

```diff
@@ -72,3 +73,12 @@ async def trigger_build_microshift(build_version: str, assembly: str, dry_run: b
             'DRY_RUN': dry_run
         }
     )
+
+
+def new_jenkins_client():
+    jenkins_server = "https://buildvm.hosts.prod.psi.bos.redhat.com:8443/"
```

Comment:
this should be in constants.py (same in existing code above) - although ideally in config we read somewhere. could change if first buildvm ever dies and we move to second one, or if we need to migrate infrastructure again... would like to minimize changes in code for that sort of thing.